src/defaults.rb

```diff
@@ -4,6 +4,11 @@
 
 
 module Metalware
+  # When new defaults are added here you may also want to add these to the
+  # option's help to be displayed.
+  # TODO: Could have these displayed in help automatically by overriding
```

Comment:
Not sure what you mean buy automatically. From the looks of it, changes to the default file will correspond to changes to help. Unless you are talking about automatically searching for a value under: `default.<action-name>.<option-name>`?